README.md

```diff
@@ -26,13 +26,29 @@ Daedalus - Cryptocurrency Wallet
    ```
    and then add the following lines:
    ```
+   build-users-group = nixbld
+
+   max-jobs = auto
+   cores = 0                                         # the builder will use all available CPU cores
+   sandbox = false
+
+   require-sigs = true
+   trusted-users = root
+   allowed-users = *
+
    substituters = https://hydra.iohk.io https://cache.nixos.org/
    trusted-substituters =
    trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
-   max-jobs = 2  # run at most two builds at once
-   cores = 0     # the builder will use all available CPU cores
-   extra-sandbox-paths = /System/Library/Frameworks
+   extra-sandbox-paths = /System/Library/Frameworks /System/Library/PrivateFrameworks /usr/lib
+
+   # If you are using MacOS, please uncomment one of the following configs, based on your chip:
+
+   # extra-platforms = x86_64-darwin                 # Mac with Intel chip
+
+   # extra-platforms = x86_64-darwin aarch64-darwin  # Mac with M1 chip
+   # system = x86_64-darwin
```

Comment:
I'm not sure we want to set `x86_64-darwin` by default on an aarch64 machine? Those should default to `aarch64-darwin`. Unless we know `aarch64-darwin` (that is native) won't work for some Daedalus stuff on M1s?

The reason is this: if you run stuff though rosetta2, ahead of time translated (x86_64 -> aarch64) binaries will end up in the rosetta2 cache (thanks to SPI you can't even see them, or delete them, unless you disable SIP). There is no mechanism for cache eviction the last time I checked, and thus building lots of x86_64 software (and running it) will slowly fill your disk with rosettas cache.

Reply:
After looking at the slack message a bit more carefully, I came to the realisation that we might intentionally set `system = x86_64-darwin` to force x86_64 Daedalus builds instead of native aarch64 ones?

Reply:
> Unless we know aarch64-darwin (that is native) won't work for some Daedalus stuff on M1s?

I'm pretty new here but from what I've heard and seen we're not ready for native `aarch64-darwin` yet. There're some ongoing processes around that. From quick investigation of our nix shell configs and haskell code I see only `"x86_64-linux" "x86_64-darwin" "x86_64-windows"` as supported platforms.

Taking above into account, I've decided to enforce `system = x86_64-darwin` in `nix.conf` to allow running nix shell on Rosetta2 (for development purposes).

@danielmain please advise if you know anything more about it.

Thank you for taking your time for review @angerman 💪🏻

Reply:
yes, that is right. We have a separated [branch](https://github.com/input-output-hk/daedalus/pull/2684) that aims to have a fully native Daedalus application.